alibi_detect/saving/schemas.py

```diff
@@ -889,6 +867,314 @@ class LearnedKernelDriftConfigResolved(DriftDetectorConfigResolved):
+    verbose: bool = True
+
+
+class LSDDDriftOnlineConfig(DriftDetectorConfig):
+    """
+    Unresolved schema for the
+    `LSDDDriftOnline <https://docs.seldon.io/projects/alibi-detect/en/stable/cd/methods/onlinelsdddrift.html>`_
+    detector.
+
+    Except for the `name` and `meta` fields, the fields match the detector's args and kwargs. Refer to the
+    :class:`~alibi_detect.cd.LSDDDriftOnline` documentation for a description of each field.
+    """
+    ert: float
+    window_size: int
+    sigma: Optional[np.ndarray] = None
+    n_bootstraps: int = 1000
+    n_kernel_centers: Optional[int] = None
+    lambda_rd_max: float = 0.2
+    device: Optional[Literal['cpu', 'cuda']] = None
+    verbose: bool = True
+
+
+class LSDDDriftOnlineConfigResolved(DriftDetectorConfigResolved):
+    """
+    Resolved schema for the
+    `LSDDDriftOnline <https://docs.seldon.io/projects/alibi-detect/en/stable/cd/methods/onlinelsdddrift.html>`_
+    detector.
+
+    Except for the `name` and `meta` fields, the fields match the detector's args and kwargs. Refer to the
+    :class:`~alibi_detect.cd.LSDDDriftOnline` documentation for a description of each field.
+    """
+    ert: float
+    window_size: int
+    sigma: Optional[np.ndarray] = None
+    n_bootstraps: int = 1000
+    n_kernel_centers: Optional[int] = None
+    lambda_rd_max: float = 0.2
+    device: Optional[Literal['cpu', 'cuda']] = None
+    verbose: bool = True
+
+
+class CVMDriftOnlineConfig(DriftDetectorConfig):
+    """
+    Unresolved schema for the
+    `CVMDriftOnline <https://docs.seldon.io/projects/alibi-detect/en/stable/cd/methods/onlinecvmdrift.html>`_
+    detector.
+
+    Except for the `name` and `meta` fields, the fields match the detector's args and kwargs. Refer to the
+    :class:`~alibi_detect.cd.CVMDriftOnline` documentation for a description of each field.
+    """
+    ert: float
+    window_sizes: List[int]
+    n_bootstraps: int = 10000
+    batch_size: int = 64
+    n_features: Optional[int] = None
+    verbose: bool = True
+
+    # validators
+    _coerce_int2list = validator('window_sizes', allow_reuse=True, pre=True)(coerce_int2list)
+
+
+class CVMDriftOnlineConfigResolved(DriftDetectorConfigResolved):
+    """
+    Resolved schema for the
+    `CVMDriftOnline <https://docs.seldon.io/projects/alibi-detect/en/stable/cd/methods/onlinecvmdrift.html>`_
+    detector.
+
+    Except for the `name` and `meta` fields, the fields match the detector's args and kwargs. Refer to the
+    :class:`~alibi_detect.cd.CVMDriftOnline` documentation for a description of each field.
+    """
+    ert: float
+    window_sizes: List[int]
+    n_bootstraps: int = 10000
+    batch_size: int = 64
+    n_features: Optional[int] = None
+    verbose: bool = True
+
+    # validators
+    _coerce_int2list = validator('window_sizes', allow_reuse=True, pre=True)(coerce_int2list)
+
+
+class FETDriftOnlineConfig(DriftDetectorConfig):
+    """
+    Unresolved schema for the
```

Comment:
Remove "unresolved" since you've done it for others..?

Reply:
https://github.com/SeldonIO/alibi-detect/pull/497#discussion_r879437444